migration-playbook.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Migrate from OpenShift SDN to OVN-Kubernetes
+  hosts: localhost
+  gather_facts: no
+  roles:
+    - role: migration
+      vars:
+        co_timeout: 1200
+        clean_migration_timeout: 120
+        change_migration_timeout: 120
+        network_type: OVNKubernetes
+        mco_update_timeout: 300s
+        mcp_completion_timeout: 18000  # Timeout in seconds
+        ovn_co_timeout: 120  # Timeout in seconds
+        ovn_multus_timeout: 300  # Timeout in seconds for waiting for Multus pods
+        verify_machine_config_timeout: 300
+        conflicting_cidr_ranges:
+          - "100.64.0.0/16"
+          - "169.254.169.0/29"
+          - "100.88.0.0/16"
+          - "fd98::/64"
+          - "fd69::/125"
+          - "fd97::/64"
+        checks:
+          - "oc wait co --all --for='condition=Available=True' --timeout=60s"
+          - "oc wait co --all --for='condition=Progressing=False' --timeout=60s"
+          - "oc wait co --all --for='condition=Degraded=False' --timeout=60s"
+          - "oc wait mcp --all --for='condition=UPDATING=False' --timeout=60s"
+          - "oc wait mcp --all --for='condition=DEGRADED=False' --timeout=60s"
+        #egress_ip: false
+        #egress_firewall: false
+        #multicast: false
+        configure_network_type: ovnKubernetes
+        #mtu: 1400
```

Comment:
note that OVN  `geneveOverhead` `1400` is based on an incorrect calculation.  [OCPBUGS-14938](https://issues.redhat.com/browse/OCPBUGS-14938)

someday it might be fixed.

Reply:
Yeah these comments are just for reference as to how they can set values.

Reply:
what value do you suggest to put as an example?

Reply:
`1400` is the standard for now.  I just commented for the record.